k8s/config/deployment.yaml

```diff
@@ -0,0 +1,47 @@
+# SPDX-License-Identifier: Apache-2.0
+# SPDX-FileCopyrightText: 2022 Satyam Bhardwaj <sabhardw@redhat.com>
+# SPDX-FileCopyrightText: 2022 Utkarsh Chaurasia <uchauras@redhat.com>
+# SPDX-FileCopyrightText: 2022 Avinal Kumar <avinkuma@redhat.com>
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#    http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: mks-server
+  namespace: mks
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: mks-server
+  template:
+    metadata:
+      labels:
+        app: mks-server
+    spec:
+      serviceAccountName: mks-server-sa
+      containers:
+      - name: mks-server
+        image: ghcr.io/ramessesii2/mks-server:latest
```

Comment:
```suggestion
        image: ghcr.io/MiniTeks/mks-server:latest
```

Reply:
until we merge this branch with main, we won't find any builds inside MiniTeks/mks-server. So, for testing use ramessesii2/mks-server:latest. Although I'm gonna change it now